src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -96,6 +94,13 @@ public sealed class ProjectInfo
         /// </summary>
         public Type HostObjectType { get; }
 
+        /// <summary>
+        /// True if project information is complete. In some workspace hosts, it is possible
+        /// a project only has partial information. In such cases, a project might not have all
+        /// information on its files or references.
```

Comment:
well, suggest name. I choose IsComplete since that is something I can come up with that the time, if you have better name, sure, no reason not to use. by the way, I has nothing tied to references.